### PR TITLE
Docs: first draft of share panel with support docs

### PR DESCRIPTION
**Pull request by chri2547**
This PR documents a new feature where customers can share panels with Grafana support and request help with troubleshooting. This feature will be available in v9.2.

### CODE REVIEW

File: docs/sources/troubleshooting/send-panel-to-grafana-support/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- xxx
+
+**To send a panel to Grafana support:**
+
+1. Open the dashboard that contains the panel you want to send to Grafana Labs.
+
+1. Hover your mouse cursor over the panel title and click **Inspect > Support snapshot**
+
+   Grafana opens a standalone support dashboard that contains the data you are sending to Grafana Labs Technical Support.
+
```
Comment (chri2547) [inline docs/sources/troubleshooting/send-panel-to-grafana-support/index.md:34]: @ryantxu , We might want to add a screenshot here of the support dashboard. If you could send me a pic, I can add it.

File: docs/sources/troubleshooting/send-panel-to-grafana-support/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+weight: 100
+---
+
+# Send a panel to Grafana support
+
+When you encounter problems with any of your visualizations, you can send the panel JSON model to Grafana Labs Technical Support and request help with troubleshooting your issue.
+
+The panel that you send includes all query response data and all visualizations settings. Upon receiving your panel, Grafana Labs Technical Support imports your data into a local version of Grafana and begins researching your problem.
+
+**Before you begin:**
```
[the remark this answers is not in the document]
Comment (ryantxu) [reply docs/sources/troubleshooting/send-panel-to-grafana-support/index.md:21]: none... just that you have grafana running and see *something* in in a panel

File: docs/sources/troubleshooting/send-panel-to-grafana-support/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+- Does the user completing this task require any permissions?
+- xxx
+- xxx
+
+**To send a panel to Grafana support:**
+
+1. Open the dashboard that contains the panel you want to send to Grafana Labs.
+
+1. Hover your mouse cursor over the panel title and click **Inspect > Support snapshot**
```
[the remark this answers is not in the document]
Comment (ryantxu) [reply docs/sources/troubleshooting/send-panel-to-grafana-support/index.md:31]: Inspect for now -- soon it should likely be something like "Get help...", but more than we are ready for in 9.2 :)

File: docs/sources/troubleshooting/send-panel-to-grafana-support/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/troubleshooting/send-panel-to-grafana-support/index.md:21]: Ok, cool. I'll remove that section.

File: docs/sources/troubleshooting/send-panel-to-grafana-support/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/troubleshooting/send-panel-to-grafana-support/index.md:23]: Ok, I'll keep it as Inspect > Support snapshot for now. Just let me know if/when it changes.

File: docs/sources/troubleshooting/send-panel-to-grafana-support/index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+   b. In the [Grafana/Grafana](https://github.com/grafana/grafana) repository, create an issue and paste the contents of the support dashboard.
+
+1. To send the panel data to Grafana Labs via a support ticket:
+
+   a. Click **Dashboard**.
+
+   Grafana downloads the support dashboard to a TXT file.
+
+b. Attach the TXT file to a support ticket that you send to [Grafana Labs Technical Support](LINK?).
```
Comment (chri2547) [inline docs/sources/troubleshooting/send-panel-to-grafana-support/index.md:39]: @ryantxu , is there a link we can add here?

**Comment by ryantxu** [comment]
Perhaps:
image


If iframes + embedding is enabled:
image


If not, much less interesting (and cluttered 🤷 )
image

**Comment by chri2547** [comment]
> Perhaps: image
> 
> If iframes + embedding is enabled: image
> 
> If not, much less interesting (and cluttered 🤷 ) image

@ryantxu , Should we tell customers to enable iframes as a prereq? Or at least tell them it's an option?

**Comment by ryantxu** [comment]
> Should we tell customers to enable iframes as a prereq? Or at least tell them it's an option?

Tell them it is an option is good, but for sure not a prereq

**Review by ryantxu — APPROVED** [review]

**Pull request merged**